### ISSUE TITLE
[Linux x64] Sqlite dependency requires GLIBC 2.14

### ISSUE THREAD

**Issue by mohameden** [sirensolutions/kibi#10]
Hello,
I'm trying to use kibi on my Linux x64 (Red Hat Enterprise Linux Server release 6.5 (Santiago)) and getting the below issue regarding Sqlite:

```
module.js:356
  Module._extensions[extension](this, filename);
                               ^
Error: /lib64/libc.so.6: version 'GLIBC_2.14' not found (required by /myPath/kibi-0.2.0-linux-x64/src/node_modules/sqlite3/lib/binding/node-v11-linux-x64/node_sqlite3.node)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/myPath/kibi-0.2.0-linux-x64/src/node_modules/sqlite3/lib/sqlite3.js:4:15)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/myPath/kibi-0.2.0-linux-x64/src/lib/sindicetech/sqliteQuery.js:5:15)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/myPath/kibi-0.2.0-linux-x64/src/lib/sindicetech/queryEngine.js:14:19)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/myPath/kibi-0.2.0-linux-x64/src/routes/proxy.js:17:19)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/myPath/kibi-0.2.0-linux-x64/src/app.js:14:13)
    at Module._compile (module.js:456:26)
```

I'm running on GLIBC 12.2 :

```
-bash-4.1$  /lib64/libc.so.6
GNU C Library stable release version 2.12, by Roland McGrath et al.
Copyright (C) 2010 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.
Compiled by GNU CC version 4.4.7 20120313 (Red Hat 4.4.7-9).
Compiled on a Linux 2.6.32 system on 2015-01-19.
Available extensions:
        The C stubs add-on version 2.1.2.
        crypt add-on version 2.1 by Michael Glad and others
        GNU Libidn by Simon Josefsson
        Native POSIX Threads Library by Ulrich Drepper et al
        BIND-8.2.3-T5B
        RT using linux kernel aio
libc ABIs: UNIQUE IFUNC
For bug reporting instructions, please see:
<http://www.gnu.org/software/libc/bugs.html>.
-bash-4.1$ yum -y update glibc
Loaded plugins: downloadonly, security
You need to be root to perform this command.
```

How should I deal with this issue? Is it possible to use another version of node-sqlite? 

Thanks a lot

**Comment by szydan** [comment]
See the thread at and try to follow the instructions 
https://groups.google.com/a/siren.solutions/forum/#!topic/kibi-users/281c2aINIrI

Shortly we will provide a compatible bindings, but I can not tell the ETA for this.

**Comment by mohameden** [comment]
Thanks a lot. I'll follow the thread : )

**Comment by szydan** [comment]
Fixed in version 0.3.0
